### PR TITLE
Feature/les 251 add plyr to project

### PR DESCRIPTION
**Pull request by valer-sky**
https://jr.it-academy.by/browse/LES-251

### CODE REVIEW

File: src/components/body/content/about/About.tsx [hunk 1]
```diff
@@ -1,4 +1,8 @@
+
+
 const About = () => {
-  return <div>About</div>;
+  return  (
+    <>About</>
```
Comment (Terapool) [inline src/components/body/content/about/About.tsx:5]: No need in react fragment here: return 'About'

File: src/components/body/content/videoplayer/Videoplayer.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    },
+  ],
+  previewThumbnails: {
+    enabled: true,
+    src: ['https://cdn.plyr.io/static/demo/thumbs/100p.vtt', 'https://cdn.plyr.io/static/demo/thumbs/240p.vtt'],
+  },
+  tooltips: {
+    controls: true,
+  },
+  //Another exemple  a videoplayer
```
Comment (Terapool) [inline src/components/body/content/videoplayer/Videoplayer.tsx:56]: Please remove comments - your code looks dirty. If you need working examples save them as javascript object in separate file or just here. Then simple switch them as props or something like this

File: src/components/body/content/videoplayer/Videoplayer.tsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    'ads',
+  ],
+  seekTime: 10,
+};
+
+// learn more https://github.com/sampotts/plyr#the-source-setter
+
+export default function App() {
+  return (
+    <>
```
Comment (Terapool) [inline src/components/body/content/videoplayer/Videoplayer.tsx:187]: No need in react fragment here

File: src/components/body/content/videoplayer/index.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.player {
+    --plyr-captions-background: rgba(0, 0, 0, 0.8); /* цвет бэкграунда субтитров*/
```
Comment (Terapool) [inline src/components/body/content/videoplayer/index.scss:2]: Please, use English comments

File: src/components/body/content/videoplayer/Videoplayer.tsx [hunk 5]
```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import Plyr from "plyr-react";
+// import Vimeo from '@u-wave/react-vimeo';
```
Comment (Terapool) [inline src/components/body/content/videoplayer/Videoplayer.tsx:3]: remove this commented code

File: src/components/body/content/videoplayer/Videoplayer.tsx [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+  );
+}
+
+export const VideoPlayer: React.FC = () => {
+  return (
+    <div className ="player">
+      <Plyr options={optionsVideoplayer}
+        source={videoSrc}
+      />
+      {/* <Vimeo options={optionsVideoplayer}
```
Comment (Terapool) [inline src/components/body/content/videoplayer/Videoplayer.tsx:199]: remove this commented code

**Review by Evgeniy121212 — CHANGES_REQUESTED** [review]
please, move subtitles files to the `/exampels/plyr/subtitles/` folder 

Also - bring newest changes from develop into that branch (currently it outdated)

File: src/components/body/content/about/About.tsx (hunk 1 above)
Comment (valer-sky) [reply src/components/body/content/about/About.tsx:5]: corrected

File: src/components/body/content/videoplayer/Videoplayer.tsx (hunk 5 above)
Comment (valer-sky) [reply src/components/body/content/videoplayer/Videoplayer.tsx:3]: corrected

File: src/components/body/content/videoplayer/Videoplayer.tsx (hunk 2 above)
Comment (valer-sky) [reply src/components/body/content/videoplayer/Videoplayer.tsx:56]: corrected

File: src/components/body/content/videoplayer/Videoplayer.tsx (hunk 3 above)
Comment (valer-sky) [reply src/components/body/content/videoplayer/Videoplayer.tsx:187]: corrected

File: src/components/body/content/videoplayer/Videoplayer.tsx (hunk 6 above)
Comment (valer-sky) [reply src/components/body/content/videoplayer/Videoplayer.tsx:199]: corrected

File: src/components/body/content/videoplayer/index.scss (hunk 4 above)
Comment (valer-sky) [reply src/components/body/content/videoplayer/index.scss:2]: corrected

**Pull request closed without merging**